### PR TITLE
doctor(Search): the code is at target; the docs and comments were not

### PR DESCRIPTION
**Pull request by peterdrier**
Section-doctor run on **Search**, 2026-09-06 (unattended). Run file: [`docs/health/runs/2026-09-06-Search.md`](https://github.com/peterdrier/Humans/blob/section-doctor/2026-09-06T061610Z/docs/health/runs/2026-09-06-Search.md). Target shape: [`Docs/health.md`](https://github.com/peterdrier/Humans/blob/section-doctor/2026-09-06T061610Z/src/Sections/Humans.Search/Docs/health.md) (new). Likely future sections: Users, Scanner, Tour, then the previously-doctored tier.

## What

First doctor pass over Search. The code already matches the target shape (one controller, one internal orchestrator over five read interfaces, one view plus one partial, no tables); the strikes are documentation, comment and test hygiene, plus one localization fix. No business behavior changes, no schema changes.

- **Docs true to the code** (findings 1–6, 8, 9): `Search.md` rewritten — the Shifts read-split has landed (`IShiftManagementServiceRead`), the nav item is `SectionNav : ISectionNav` not a `_Layout` link, nobodies-collective/Humans#993 and #998 are closed as designed everywhere, every line-number/count citation dropped, history narration cut. Same corrections in `Contracts/README.md`, `data-access.md` (Shifts is DB-backed) and `docs/features/global/global-search.md`; a citation to a non-existent memory atom removed.
- **Comments and csproj** (findings 7, 14, 15): the 45-line `ISearchService` summary and a dozen smaller comments now say what each type is and point at `Docs/Search.md`; two unneeded implicit usings and an empty `ItemGroup` dropped; the Events gate parenthesised so it reads as it parses.
- **Tests** (findings 10–13): `SectionNavTests` pins the authenticated-only nav gate; the length-gate and unbounded-cap tests now assert the Events section; the human pass-through test renamed to what it pins (the second-opinion reviewer rejected deleting it); unused NodaTime references dropped.
- **Localization** (finding 16): the two hardcoded English `aria-label`s on `/Search` now read `Search_QueryAriaLabel` / `Search_FilterAriaLabel` from `SearchResource` in all six cultures.
- **Bookkeeping:** `Docs/health.md` (target shape), the run file, and the sweep commit applying queued items from merged run files. The central debt-ledger entry this run had filed for the unpinned view-side `Features:Events` gate (finding 17) was deleted in round 5 — nobodies-collective/Humans#1182 is that debt's record, and a ledger line pointing at an issue is a duplicate.

## Why

The section's prose recorded three things as pending that had shipped, and two rulings as open that had closed. A reader working from the docs would have reintroduced the read-split or "fixed" a settled destination-page decision.

## Existing surface checked

No new durable surface. The two aria-label keys join the existing `SearchResource` set, which already owned every `/Search` string; the nav test uses `ISectionNav`'s existing `Visible` predicate.

## UI changes / screenshots

`aria-label` text on the `/Search` query box and filter chip group is now localized. Build-verified and parity-tested; not rendered live (the container has no database). The preview deploy is the place to see it; no visible change in English.

## Checklist

- ~~**Section labeled**~~ — doctor run; no section label applied by the run.
- [x] **Targeting `main` on `peterdrier/Humans`** (the QA fork).
- [x] **Branched off `origin/main`** (`10199a23`), not `upstream/main`; `main` merged in 2026-09-14.
- [x] **Issue refs are qualified** (`owner/repo#N`); one malformed `nobodies-collective#987` ref fixed on the way, and a bare second ref in `health.md` qualified in round 5.
- ~~**EF migrations**~~ — none; no schema change.
- [x] **NuGet packages updated?** No additions; two unused test-only references removed, which the About page does not list.
- ~~**New project rule?**~~ — none.
- [x] **Reuse-first checked** — no new public types, endpoints, DTOs or DI registrations.
- [x] **Build + test pass locally**: `dotnet build Humans.slnx -v quiet`, `dotnet test Humans.slnx -v quiet` (46 suites green), `dotnet format whitespace Humans.slnx --verify-no-changes` clean — re-run after the merge.
- [x] **Nav coverage** — no new page.
- [x] **No magic strings** — resx keys only.
- [x] **Dates/times via NodaTime**, icons via Font Awesome 6 — no dates touched; icons unchanged.

## Reviewer notes

The reviewer gate ran once (finding 12) and rejected a deletion; the rename it accepted is what shipped. Selector defect found on the way: peterdrier/Humans#1601.

All five Codex review rounds are now spent. Rounds 1–4: a stale key count and a `Policy`-absence assertion dropped; finding 17 re-routed from the section `debt.yml` to the central ledger; set-size counts removed from `health.md`; `SearchResource` ownership described by renderer instead of a prefix list the new aria-label key did not match. Round 5 (P1): that central ledger entry deleted outright, because nobodies-collective/Humans#1182 now records the same debt and `debt-ledger-additions` treats an issue as a complete record; the P2 bare-ref fix rode along at no extra CI cost. One finding declined (the Agent ledger's reforge figures are a merged run file's queued item applied verbatim). The budget is exhausted — any further finding stops the run and surfaces instead of being fixed unattended.

**2026-09-14:** Peter answered the Needs-Peter queue and `main` was merged in. The merge conflicted in four files, all of them this run's sweep commit: later runs had already applied the same queued items to `dependency-graph.md`, `debt-ledger-additions.md` and Users' `debt.yml`, so `main`'s wording was taken for all three and this run's versions dropped out of the diff. The merge also surfaced a duplicate `classDef monitor` in `dependency-graph.md` (two runs each styled the Monitor carve-out), now recorded in the ledger inbox. Neither the merge nor Peter's answers spend review rounds.

## Needs Peter

Answered 2026-09-14 except where noted.

- [x] 12 — delete the human match-context pass-through test? **Leave it.** No change.
- [x] 17 — pin the view-side `Features:Events` gate with a `Humans.Web.Tests` render case? **Neither option.** Searchable targets become a DI-discovered fanout, so a switched-off section never registers and there is nothing to skip or hide. Filed as nobodies-collective/Humans#1182; the ledger entry this run had added was deleted in round 5, since the issue is the record.
- [x] 23 — treat a section absent from reforge's compact output as score 0? **Yes.** peterdrier/Humans#1601 was closed as completed but the selector still maps a missing section to `sys.maxsize`, so it is reopened.
- [ ] 24 — does `SearchPageRenderTests.An_anonymous_visitor_cannot_reach_search` belong in Search's own test project rather than `Humans.Integration.Tests`? The end-to-end redirect needs a running host, but the half Search owns (`SearchController` carries `[Authorize]`) is CI-pinnable in `tests/Humans.Search.Tests` and is not pinned today.
- [ ] 25 — Phase 5: a `debt:` sweep item must name the check that proves it absent, not only the target file?

🤖 Generated with [Claude Code](https://claude.com/claude-code)

https://claude.ai/code/session_01NMhJMdY5Ks1sRf21wo3aaX

### CODE REVIEW

**Comment by peterdrier** [comment]
| Component | Phase | Model | Fresh in | Out | Cache write | Cache read | ~$ |
|---|---|---|---|---|---|---|---|
| worktree | phase1 | fable | 512 | 11,714 | 120,054 | 1,824,095 | 3.92 |
| select section | phase2 | fable | 96 | 3,366 | 7,584 | 419,214 | 0.68 |
| assess | phase3 | fable | 1,880 | 201,505 | 649,420 | 13,641,018 | 31.85 |
| Freshness (subagent) | phase3 | opus | 38 | 8,314 | 160,309 | 584,753 | 1.50 |
| Tests (subagent) | phase3 | opus | 38 | 1,491 | 136,962 | 552,689 | 1.17 |
| History (subagent) | phase3 | opus | 74 | 2,208 | 285,938 | 1,627,475 | 2.66 |
| Comments (subagent) | phase3 | opus | 58 | 1,462 | 187,041 | 923,137 | 1.67 |
| Prose & surface (subagent) | phase3 | haiku | 632 | 2,452 | 309,581 | 4,415,496 | 0.84 |
| strike: target shape commit | phase4 | fable | 34 | 3,809 | 28,959 | 334,492 | 0.89 |
| strike: docs truth (read-split, rulings, nav, citations) | phase4 | fable | 1,314 | 53,401 | 317,313 | 4,709,668 | 11.36 |
| strike: comment and csproj cuts (ISearchService, Section, SearchResource, SectionNav, csproj, DTOs, controller, service, views, tests) | phase4 | fable | 1,344 | 232,516 | 193,734 | 6,788,417 | 20.85 |
| abb5d5ab732751eca (subagent) | phase4 | fable | 1,012 | 3,425 | 264,331 | 1,281,425 | 4.77 |
| strike: tests (nav gate, Events assertions, NodaTime refs) | phase4 | fable | 422 | 12,441 | 29,886 | 2,997,217 | 4.00 |
| strike: in-section debt.yml | phase4 | fable | 256 | 6,170 | 19,016 | 1,549,601 | 2.10 |
| bookkeeping | phase5 | fable | 1,412 | 89,275 | 153,221 | 10,998,559 | 17.39 |
| retro | phase6 | fable | 332 | 8,681 | 11,023 | 3,536,573 | 4.11 |
| PR | phase7 | fable | 32 | 304 | 269 | 274,803 | 0.29 |
| **total** | | | 9,486 | 642,534 | 2,874,641 | 56,458,632 | **110.04** |

API-equivalent $, list rates; run under subscription quota. Measured Phase 1 to PR creation; PR create/backfill and Phase 8 excluded.

Peak main-thread context: 288,860 tokens (assess).
Compactions detected: 1 (strike: target shape commit).

The unnamed phase-4 subagent row is the second-opinion reviewer (finding 12). The aria-label strike shares a mark timestamp with the tests strike and is folded into that row.

---
_Generated by [Claude Code](https://claude.ai/code)_

File: tests/Humans.Search.Tests/SectionNavTests.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+/// </summary>
+public sealed class SectionNavTests
+{
+    [HumansFact]
+    public void TheSearchItem_ShowsToAnAuthenticatedUser_WithNoProfileOrPolicy()
+    {
+        var item = new SectionNav().Items().Should().ContainSingle().Subject;
+        var signedIn = new ClaimsPrincipal(new ClaimsIdentity(authenticationType: "test"));
+
+        item.Policy.Should().BeNull();
```
[the remark this answers is not in the document]
Comment (peterdrier) [reply tests/Humans.Search.Tests/SectionNavTests.cs]: Fixed in fd70a3ea — the `Policy.Should().BeNull()` line is gone; the two `Visible` predicate tests stay. Not adding a `SectionNavViewComponent` test here: that composition (Visible then Policy, null policy passes) is the Shell's and is already pinned by `SectionSeamTests` in `Humans.Web.Tests`.

---
_Generated by [Claude Code](https://claude.ai/code)_

File: src/Sections/Humans.Search/Docs/health.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  switching on `Type`.
+
+## Load-bearing weirdness
+
+- **Whole-project references to the five publishers**, not just their leaves: `@addTagHelper`
+  binds only against a referenced assembly. An orchestrator referencing what it orchestrates
+  is expected; only cycles are forbidden.
+- **`_ViewImports` carries one `@addTagHelper` per publisher.** Dropping one ships that
+  bucket as inert literal markup on a green build; the integration render tests are the guard.
+- **Two localizers in the view**: the section's own 12 keys, plus `SharedResource` for
```
[the remark this answers is not in the document]
Comment (peterdrier) [reply src/Sections/Humans.Search/Docs/health.md]: Fixed in fd70a3ea — the count is dropped; the line now names `SearchResource` without a number.

---
_Generated by [Claude Code](https://claude.ai/code)_

File: src/Sections/Humans.Search/Docs/debt.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Section debt ledger for Humans.Search. Shape + routing: memory/process/debt-ledger-additions.md.
+# /debt-sweep pools every src/Sections/*/Docs/debt.yml into the same inbox as the central ledger.
+version: 1
+inbox:
+  - added: 2026-09-06
+    what: "View-side half of the Features:Events gate is unpinned in CI (finding 17, /section-doctor on Search 2026-09-06): SearchServiceTests pins that the service never calls Events when the flag is off, but nothing CI-run asserts that Index.cshtml hides the Events chip and heading — the only render tests (SearchPageRenderTests, GlobalSearchSectionRenderTests) live in the local-only Humans.Integration.Tests. Candidate: a Web.Tests page-render case with Features:Events=false asserting the chip is absent."
```
[the remark this answers is not in the document]
Comment (peterdrier) [reply src/Sections/Humans.Search/Docs/debt.yml]: Fixed in 4e15d114 — moved to the `inbox:` of `docs/architecture/debt-ledger.yml`. The fix is a CI render case and `Humans.Web.Tests` has no page-render harness, so it is shared test infrastructure, not Search code; the routing atom sends that central. The section `debt.yml` held nothing else and is removed; the run file records the move.

---
_Generated by [Claude Code](https://claude.ai/code)_

File: src/Sections/Humans.Search/Docs/health.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+The shapes imply exactly today's layout, and nothing more:
+
+- **One controller, one action** (`GET /Search?q=&filter=`), which sorts and assembles
+  the view-model — the only logic it holds.
+- **One internal orchestrator** behind `ISearchService`, one method: trim, gate, fan out,
+  carry each section's key/score through untouched.
+- **One results record, one view-model, one view plus one partial** that switches a row
+  onto its owner's component.
+- **A member-nav item, a resource marker with six resx files, `_ViewImports`** binding
+  five publishers' tag helpers.
```
[the remark this answers is not in the document]
Comment (peterdrier) [reply src/Sections/Humans.Search/Docs/health.md]: Fixed in 79a290c9 — every set-size count in `health.md` is gone (publishers, buckets, kinds, resx files), replaced by defining predicates. The structural "one controller, one action, one view plus one partial" lines stay: they are the target's prescription, not a tally of something the code owns. The `five`/`four` wording in `Search.md` and `global-search.md` predates this PR on main and stays per the atom's review-scope paragraph.

---
_Generated by [Claude Code](https://claude.ai/code)_

File: src/Sections/Humans.Agent/Docs/debt.yml [hunk 5]
```diff
@@ -5,3 +5,6 @@ inbox:
   - added: 2026-08-28
     what: "Test gaps from the invariant matrix (finding 14, /section-doctor on Agent 2026-08-28): (a) enabled gate — no test asserts the widget hides when Enabled=false (the other half, POST /Agent/Ask → 503, is pinned by AgentControllerTests.Ask_accepts_a_message_at_exactly_the_cap); (b) DailyTokenCap is set in every fixture but never exceeded — the third cap of invariant 3 has no reject test; (c) anonymous → 401 unpinned anywhere; (d) nothing asserts the tool catalog AgentPromptAssembler advertises equals AgentToolNames.All; (e) AgentFeatureSpecReader has no path-traversal negative test (CommunityFaqReader does); (f) an abuse-flagged message is never driven through AskAsync to a RefusalReason row."
     review: light
+  - added: 2026-08-28
+    what: "AgentService.RunTurnAsync is 210 LOC / CC 42 and AnthropicClient.StreamAsync 153 LOC / CC 42 (reforge). Both are the load-bearing streaming loops the target shape blesses (manual enumerator for error-path billing; SSE block assembly); recorded so the score is not mistaken for unexamined debt (/section-doctor on Agent 2026-08-28)."
```
[the remark this answers is not in the document]
Comment (peterdrier) [reply src/Sections/Humans.Agent/Docs/debt.yml:9]: Not changing. This entry is the sweep commit applying a queued item from the merged Agent run file (`docs/health/runs/2026-08-28-Agent.md`) verbatim, which is what the sweep does; it is not new prose from this PR. The figures are a dated reforge measurement recorded as the finding, not a doc claim about the code, and the Agent section's ledger is where its author put them. Rewording another section's queued item is out of this PR's scope; P2 in round 3 is also below the bar.

---
_Generated by [Claude Code](https://claude.ai/code)_

File: src/Sections/Humans.Search/Docs/Search.md [hunk 6]
```diff
@@ -20,105 +21,94 @@
...
+- **Decorator decision — no caching decorator.** Humans, Teams, Camps and Events are served from their owning sections' warm in-memory snapshots (`CachingUserService`, `CachingTeamService`, `CachingCampService`, `CachingEventService` — the last bound to `IEventServiceRead` in Events' `Section.cs`). **Shifts is the only DB-backed bucket:** `ShiftManagementService.SearchAsync` runs a case-insensitive Postgres `ILike` for text and a rota lookup for a GUID. A Search-level cache would cache a cache and duplicate invalidation the owners already do.
+- **Resource set — split by renderer.** `SearchResource` owns the `Search_Filter*` and `Search_Global*` keys: all of `/Search`'s own copy. Keys with the `Search_` prefix whose renderer is elsewhere stay in `SharedResource`: `Search_Title`/`Search_Placeholder` belong to `/Profile/Search`, `Search_NoResults`/`Search_MatchedIn` to `Humans.Users`, and `Search_MinChars` is rendered by `/Profile/Search` **and** by this section's `Index.cshtml` — so the section binds `SharedLocalizer` for that one call rather than splitting the person-search page's message set. A prefix is not an owner.
```
[the remark this answers is not in the document]
Comment (peterdrier) [reply src/Sections/Humans.Search/Docs/Search.md]: Fixed in 7afb6a0d — ownership is now described by renderer ("every key that only `/Search` renders") in both `Search.md` and the `SearchResource` summary; the prefix list is gone, which is what the same bullet's "a prefix is not an owner" already concluded.

---
_Generated by [Claude Code](https://claude.ai/code)_

File: docs/architecture/debt-ledger.yml [hunk 7]
```diff
@@ -267,6 +267,9 @@ themes:
     notes: Each inbox entry carries its own review tier; drained entries are deleted from the list.
 inbox:
   # Seeded from the open items in docs/architecture/tech-debt-2026-04-23.md.
+  - added: 2026-09-06
+    what: "View-side half of Search's Features:Events gate is unpinned in CI: SearchServiceTests pins that the service never calls Events when the flag is off, but nothing CI-run asserts that Search's Index.cshtml hides the Events chip and heading — the only render tests (SearchPageRenderTests, GlobalSearchSectionRenderTests) live in the local-only Humans.Integration.Tests, and Humans.Web.Tests has no page-render harness to add a case to. Routed here rather than to Search's section ledger because the fix is shared test infrastructure, not Search code (finding 17, /section-doctor on Search 2026-09-06; moved on a review finding on peterdrier/Humans#1602)."
+    review: panel
```
[the remark this answers is not in the document]
Comment (peterdrier) [reply docs/architecture/debt-ledger.yml]: Fixed in 4848dd10 — the entry is deleted, not rerouted again.

You are right on the framing: `Humans.Integration.Tests` is local-only by design and its absence from CI is never a finding, so an entry premised on that was going to send a future sweep after infrastructure the testing policy forbids.

The entry is also redundant now. Peter ruled on 2026-09-14 that the real fix is making searchable targets a DI-discovered fanout, so a switched-off section never registers and the `Features:Events` double gate disappears rather than being pinned. That is filed as nobodies-collective/Humans#1182, and `debt-ledger-additions` counts a GitHub issue as a complete debt record on its own. The run file's finding 17 row now points at the issue and says why the ledger entry went.

---
_Generated by [Claude Code](https://claude.ai/code)_

File: src/Sections/Humans.Search/Docs/health.md [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+- Anonymous cannot reach `/Search`; the nav item shows to any authenticated user, profile
+  or not.
+
+## 5. Seams
+
+- **Privileged search** (nobodies-collective/Humans#693): if it comes, it is a per-bucket
+  scope, not a global enum — the field mask constant and the per-section filter calls are
+  where it would plug in. Reserved, not built.
+- **Rota reach from a past event** and **`/Camps/{slug}` season gating**: both ruled
+  destination-page matters (nobodies-collective/Humans#993, #998, closed as designed).
```
[the remark this answers is not in the document]
Comment (peterdrier) [reply src/Sections/Humans.Search/Docs/health.md]: Fixed in 4848dd10 — both spellings are now `nobodies-collective/Humans#998`, in `health.md` and in the run file's finding 2 row. You were right that the run file carried the same shorthand.

A P2 in round five is below the bar this PR is triaging at, but the round-five commit was already being made for the P1 on the ledger entry, so this rode along at no extra cost. A repo-wide scan of the files this PR authors finds no other bare reference.

---
_Generated by [Claude Code](https://claude.ai/code)_
